docs/Updating-v0.3.md

```diff
@@ -0,0 +1,24 @@
+# Migrating from v0.2 to v0.3
```

Comment:
Presumably this is also for those on v0.1?
- Change title to Migrating to v0.3
- Change file name to Migrating-to-v0.3.md